### PR TITLE
Add pydarm PyPI package to conda-forge

### PR DESCRIPTION
**Pull request by eagoetz**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/pydarm/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  imports:
+    - pydarm
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/pydarm/
+  summary: "'pyDARM modeling package for LIGO detector calibration'"
```
Comment (duncanmmacleod) [inline recipes/pydarm/meta.yaml:45]:
```suggestion
  summary: pyDARM modeling package for LIGO detector calibration
```

File: recipes/pydarm/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  license: GPL-3.0
```
Comment (duncanmmacleod) [inline recipes/pydarm/meta.yaml:46]: This statement is ambiguous, either it should be 

```suggestion
  license: GPL-3.0-only
```

or

```suggestion
  license: GPL-3.0-or-later
```

**Comment by duncanmmacleod** [comment]
> Just to confirm about the `pytest` checks: do you want me to make this and the other upstream changes before this is merged? Or do you have a different plan in mind? 

That's really up to you, the current recipe works fine, so it would be an enhancement to run the pytest suite, which can be added later.

**Comment by eagoetz** [comment]
@conda-forge/staged-recipes, this recipe is ready for review, thanks

**Pull request merged**